### ISSUE TITLE
Technique 'DeferredRendererPBR' compile error

### ISSUE THREAD

**Issue by htlpt** [electronicarts/gigi#2]
main.gg compiles fine in Editor, but 'file not found' in Viewer
![image](https://github.com/user-attachments/assets/e708c460-1217-4f52-b972-0d855a1853b5)

Octahedral.hlsl is in the folder though
![image](https://github.com/user-attachments/assets/0b6a8b27-e7e6-4462-bcca-a0f78402088e)

**Comment by Atrix256** [comment]
That's strange, I'm not able to reproduce this myself.
Did you build the code yourself, or did you use the binary package in the release (if so, .zip or the installer?)
Do you have any changes to the code or techniques locally?
Does this happen 100% of the time?

Trying to think what the issue might be:
* I wonder if your anti virus could be preventing the open for read done by the shader compiler?
* I thought perhaps the file copies were non blocking, and so the file read could be happening before the write was done, but I just verified that the copies are blocking (fopen, fwrite, fclose).  These copies happen in RunBackend_Interpreter(), in Backend_Interpreter.cpp.

**Comment by Atrix256** [comment]
A shot in the dark is to try running Gigi in administrator mode to rule out any file permission issues.

**Comment by Atrix256** [comment]
Another shot in the dark to try would be to make sure your graphics driver is up to date, and any pending "windows updates" are done, in case they are affecting the shader compiler logic.

**Comment by htlpt** [comment]
I'm building from source on Windows 11, other examples work just fine. 
I've also tried administrator mode and the latest prebuilt release 0.99.1, still only this example has error (100% on my pc).
And I just noticed only in this sample the .gg file got upgraded, and shader compiler changed from FXC to DXC
![image](https://github.com/user-attachments/assets/b7bd9b91-9878-4b68-8185-a3f924b18e89)
the .gg file seems to be in older version 0.93b, other examples are in 0.98b, could that be the problem?

I also tried setting main.gg and all its subgraphs back to FXC and setting *_6_1 back to *_5_1, doesn't seems to work either

**Comment by mapmapbear** [comment]
Is there a solution to this problem? I've met it, too

**Comment by Atrix256** [comment]
No, not yet. I've not been able to reproduce it at all.  Could you paste the error you get like the original post? I'm curious where your temp folder is pointing. Thanks!

**Comment by mapmapbear** [comment]
> No, not yet. I've not been able to reproduce it at all. Could you paste the error you get like the original post? I'm curious where your temp folder is pointing. Thanks!

It appears in any Gigi, whether it's the master branch or the release version, in my
![image](https://github.com/user-attachments/assets/f4871e3f-c14e-4948-957b-be9e8f168886)
